python_modules/libraries/dagstermill/dagstermill/io_managers.py

```diff
@@ -49,7 +57,7 @@ def handle_output(self, context: OutputContext, obj: bytes):
         # the output notebook itself is stored at output_file_path
         output_notebook_path = self._get_path(context)
         mkdir_p(os.path.dirname(output_notebook_path))
-        with open(output_notebook_path, self.write_mode) as dest_file_obj:
+        with open(output_notebook_path, WRITE_MODE) as dest_file_obj:
```

Comment:
self.write_mode and self_read_mode are still set as instance vars, do we want to remove those or remove these global var versions?